js/components/Feed.jsx

```diff
@@ -157,18 +158,12 @@ var Feed = React.createClass({
     var newPagination = this.state.pagination + PAGINATION_SIZE;
 
     // If we're on the end of the pagination, we need to fetch more transactions
-    if (this.state.pagination + PAGINATION_SIZE >= this.state.transactions.length) {
+    // Otherwise, we know we can flip to another page, since the button was shown in the first place
+    if (newPagination >= this.state.transactions.length) {
       // Get the date of the last transaction in the list
       // Set the new timestamp to one millisecond earlier than that
       var newTimeStamp = new Date(new Date(this.state.transactions[this.state.transactions.length-1].timeStamp)-1);
       this.getTransactions(this.state.filter, newTimeStamp, this.props.category);
-
```

Comment:
The show older method no longer needs to check if we are at the end of the list. If we ever hit the end through our ajax calls, then the final index was set